modules/gcp/jumpbox/gcp-cleanup.sh

```diff
@@ -3,4 +3,25 @@
 export PROJECT=$1
 
 echo 'Destroying K8s Stale PVCs...'
-for pvc in $(gcloud compute disks list --project $PROJECT --filter="name:pvc*" --uri);do echo "Removing $pvc..."; gcloud compute disks delete $pvc --quiet; done
\ No newline at end of file
+for pvc in $(gcloud compute disks list --project $PROJECT --filter="name:pvc*" --uri);do echo "Removing $pvc..."; gcloud compute disks delete $pvc --quiet; done
+
+# Get a list of all managed zones in the project
+ZONES=$(gcloud dns managed-zones list --project="$PROJECT" --format="value(name)")
```

Comment:
Do we really need this?
If this about destroying the cluster zones, why simply destroying the zone via terraform is not enough?

Reply:
GCP APIs don't allow the zone deletion with the existing DNS records.